ynput/ayon-third-party
# Chore: Update functions

## Changelog Description
Revisited the client code after some time and fix/enhance cobase.

## Additional info
Use new function `get_launcher_storage_dir` instead of `get_ayon_appdirs`.

Don't use code from `ayon_common` to download and extract files. I don't understand how it could work? `ayon_common` is available only in AYON launcher.

### Notes
We might move some of the helper functions to calculate hash and unarchive file to ayon core.

## Testing notes:
1. Make sure to remove existing addon versions from `%LOCALAPPDATA%/Ynput/AYON/addon/`.
2. Create package, upload it to server, use it in bundle.
3. Run some publishing.
